charts/netdata/templates/_helpers.tpl

```diff
@@ -38,6 +38,17 @@ Return the target Kubernetes version
 {{- default .Capabilities.KubeVersion.Version .Values.kubeVersion -}}
 {{- end -}}
 
+{{/*
+The function determines if the Netdata version is less than 3.0-0.
+*/}}
+{{- define "netdata.verLess3" -}}
+{{- if semverCompare "<3.0-0" .Chart.AppVersion -}}
```

Comment:
This will change if we decide not to do a major release. See [this Slack discussion](https://netdata-cloud.slack.com/archives/CNLAM2B19/p1735663645012739) for details.